### PR TITLE
Improve the Chinese translation.

### PR DESCRIPTION
**Pull request by lndj**
5 seconds from now.
In Chinese,it should be "距现在5秒",not "5秒距现在"。

### CODE REVIEW

**Comment by lucasmichot** [comment]
Can you also add tests @lndj ?

File: tests/Localization/zhTest.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class ZhTest extends AbstractTestCase
+{
+    public function testDiffForHumansLocalizedInTurkish()
+    {
+        Carbon::setLocale('zh');
+
+        $scope = $this;
+        $this->wrapWithTestNow(function () use ($scope) {
+            Carbon::now()->addSeconds(5)->diffForHumans();
```
Comment (lucasmichot) [inline tests/Localization/zhTest.php]: Not sure I understand what you are testing there?

**Comment by lucasmichot** [comment]
Ping @lndj

**Comment by lndj** [comment]
sorry,so busy recently.tomorrow.

**Comment by lucasmichot** [comment]
No hurries ;-)

**Comment by lndj** [comment]
E...I just wrote test for the section of I changed~

**Comment by lucasmichot** [comment]
For all would be better
Feel free to copy paste the french tests , change locale to zh, and modify
the expectations

File: src/Carbon/Lang/zh.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
  */
 return array(
     'year'      => ':count年',
-    'month'     => ':count月',
+    'month'     => ':count个月',
```
Comment (lndj) [inline src/Carbon/Lang/zh.php]: We usually call  '5个月前', not '5月前'. These are two totally different wording.

Comment (lucasmichot) [inline src/Carbon/Lang/zh.php]: I totally trust you @lndj !
Can you apply https://styleci.io/analyses/q1bPvv
And squash you commits ?

**Comment by lndj** [comment]
Sure.But it is time to bed in China.
Later.

**Comment by lucasmichot** [comment]
Sure @lndj 
You can just merge https://github.com/lndj/Carbon/pull/1 if you want
Thx for all and good night

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

**Comment by lndj** [comment]
Help! 
About StyleCI.
(^_^)

File: tests/Localization/zhTest.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?php
```
Comment (lucasmichot) [inline tests/Localization/zhTest.php]: Remove this file

**Comment by lndj** [comment]
When you merge it ,please update the packagist.Thanks~

**Pull request merged**